submissions/BlueLamantine/js-dom/dom.js

```diff
@@ -0,0 +1,154 @@
+const COFFEE = [{
```

Comment:
I appreciate you approach with creating this `COFFEE` array, but I'd recommend splitting your data from the user actions. It won't be connected. I mean the `state` attribute. It's better to move this state outside from your data, because it isn't consistent enough

Reply:
Let me explain. This attribute is a marker for main elements visibility, when DOM generated at first time. As you recommended I removed it from data and instead of ```state``` I use attribute ```id``` for this role (main element visible after reload page).